packages/oui-message/error_l.svg

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
There is a summary of what Felix told be about that: 
In the next Sketch update svg should be optomized by default and should be really better. Until then, we should use Illustrator to generate our svg. What you see now is the output without hand modification.